src/result/pending.ts

```diff
@@ -0,0 +1,178 @@
+import returnThis from '@utils/return-this';
+import Exception from '@utils/exception';
+
+import { pipe, property } from '@utils/utils';
+import { Optional } from '@optional/index';
+import { Result } from '@generated/result.generated';
+import { AsyncResult } from '@generated/asyncResult.generated';
+
+import { IResultCallbacks } from '@src/result.types';
+import { isAsyncResult } from './result.utils';
+
+export class PendingClass<T> extends AsyncResult<T> {
+
```

Comment:
Again, pretty minor, but there's an extra newline here

Reply:
Yeah, there are still a lot of left over new-lines, i'll go through the project and remove unnecessary code.